package.json

```diff
@@ -28,25 +26,17 @@
     "test:unit": "mocha test/unit/**/*.test.ts",
     "download-spec-tests": "node -r ts-node/register test/spec/downloadTests.ts"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/ChainSafe/blst-ts.git"
-  },
-  "keywords": [],
-  "author": "ChainSafe Systems",
-  "license": "Apache-2.0",
-  "bugs": {
-    "url": "https://github.com/ChainSafe/blst-ts/issues"
+  "dependencies": {
+    "node-addon-api": "^6.1.0",
+    "node-gyp": "^10.0.1",
+    "ts-node": "^10.9.2"
```

Comment:
See note above, this will be moved back to devDependencies in an upcoming PR.  For now it doesn’t do much hard as most TS repos will have it installed anyway…